spec/services/email_manager/donation_receipt_sender_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+describe EmailManager::DonationReceiptSender, '#call' do
+  let(:payment_intent) { create :payment_intent }
+  let(:receipt_params) do {
+      amount: 5000,
+      merchant: "Leaky Cauldron",
+      payment_intent: payment_intent,
+  }
+  end
+
+  it 'should call donation email sender methods' do
+    sender = class_double("EmailManager::Sender")
```

Comment:
this has some weird line breaks — mind running rubocop -a on your changes? non-blocking